### ISSUE TITLE
[WH] Declare Anathema

### ISSUE THREAD

**Issue by MKorin** [WarEmu/WarBugs#11884]
Since change preventing use DA in the air skill dont turn off collision so you can be blocked by enemy. On video collision dont let me jump straight backward.
https://youtu.be/rl04S_fDscw

Another issue is when you use it in air ability fires but it doesnt work but goes on cooldown.

**Comment by sioding** [comment]
Next time please only one  issue per report - but as far as i have seen it the second one is fixed with next patch.
